spec/brainstem-model-spec.coffee

```diff
@@ -4,6 +4,109 @@ describe 'Brainstem.Model', ->
   beforeEach ->
     base.data.reset()
 
+  describe 'instantiation', ->
+    newModel = null
+
+    itReturnsCachedInstance = ->
+      it 'returns cached instance', ->
+        expect(newModel).toEqual model
```

Comment:
It accomplishes the same thing. `cid` would probably be plenty sufficient but the whole point of this PR is to ensure a reference so I think in this case I prefer actually asserting the reference.